src/main/java/com/todoist/pojo/EventExtraData.kt

```diff
@@ -14,5 +14,6 @@ open class EventExtraData(
     open var parentProjectName: String?,
     open var parentProjectColor: Int?,
     open var parentItemContent: String?,
-    open var noteCount: Int?
+    open var noteCount: Int?,
+    open var subTasksReset: Int?
```

Comment:
Nit: Let's add the trailing comma? 😊

Reply:
Originally I didn't add it to not create inconsistency in the project (since almost all classes don't have trailing commas) and probably it's better to update code styles in single commit. But since you're pointing to it, added it here 😊